python/vyos/utils/process.py

```diff
@@ -20,10 +20,23 @@
 from subprocess import STDOUT
 from subprocess import DEVNULL
 
+
+def get_wrapper(vrf, netns, auth):
+    wrapper = ''
+    if vrf:
+        wrapper = f'ip vrf exec {vrf} '
+    elif netns:
+        wrapper = f'ip netns exec {netns} '
+    if auth:
```

Comment:
What does auth do?

Reply:
auth was added in this  [PR](https://github.com/vyos/vyos-1x/pull/4225#pullrequestreview-2494419628) to prevent showing username and password in error message